Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -141,11 +141,18 @@ public override IEnumerable<Guid> ListChainIds()
         /// <inheritdoc/>
         public override void DeleteChainId(Guid chainId)
         {
-            _lastStateRefCaches.Remove(chainId);
+            try
+            {
+                _lastStateRefCaches.Remove(chainId);
 
-            var cfName = chainId.ToString();
-            _chainDb.DropColumnFamily(cfName);
-            _stateRefDb.DropColumnFamily(cfName);
+                var cfName = chainId.ToString();
+                _chainDb.DropColumnFamily(cfName);
+                _stateRefDb.DropColumnFamily(cfName);
```

Comment:
How about catching the exception individually in `_chainDb` and `_stateRefDb`?

Reply:
As apparently `DropColumnFamily()` method also throws the same exception, I wrap all of them in a single `try` block.  FYI:

```
Unhandled Exception: System.Collections.Generic.KeyNotFoundException: The given key 'a7660f0c-d8a0-4550-8ffe-e725f0949395' was not present in the dictionar\
y.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at RocksDbSharp.RocksDb.DropColumnFamily(String name)
   at Libplanet.RocksDBStore.RocksDBStore.DeleteChainId(Guid chainId)
   at Libplanet.Net.Swarm`1.PreloadAsync(Nullable`1 dialTimeout, IProgress`1 progress, IImmutableSet`1 trustedStateValidators, EventHandler`1 blockDownload\
Failed, CancellationToken cancellationToken)
   at Libplanet.Standalone.Hosting.LibplanetNodeService`1.StartAsync(CancellationToken cancellationToken) in /app/Libplanet.Standalone/Hosting/LibplanetNod\
eService.cs:line 123
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at NineChronicles.Standalone.NineChroniclesNodeService.Run(CancellationToken cancellationToken) in /app/NineChronicles.Standalone/NineChroniclesNodeServ\
ice.cs:line 119
   at NineChronicles.Standalone.Executable.Program.Run(String appProtocolVersionToken, String genesisBlockPath, Boolean noMiner, String host, Nullable`1 po\
rt, Int32 minimumDifficulty, String privateKeyString, String storeType, String storePath, String[] iceServerStrings, String[] peerStrings, Boolean noTruste\
dStateValidators, String[] trustedAppProtocolVersionSigners, Boolean rpcServer, String rpcListenHost, Nullable`1 rpcListenPort) in /app/NineChronicles.Stan\
dalone.Executable/Program.cs:line 129
   at Cocona.Command.Dispatcher.Middlewares.CoconaCommandInvokeMiddleware.DispatchAsync(CommandDispatchContext ctx)
   at Cocona.Command.Dispatcher.Middlewares.HandleParameterBindExceptionMiddleware.DispatchAsync(CommandDispatchContext ctx)
   at Cocona.Command.Dispatcher.Middlewares.HandleExceptionAndExitMiddleware.DispatchAsync(CommandDispatchContext ctx)
   at Cocona.Command.Dispatcher.CoconaCommandDispatcher.DispatchAsync(CancellationToken cancellationToken)
   at Cocona.Lite.Hosting.CoconaLiteAppHost.RunAsyncCore(CancellationToken cancellationToken)
```

Reply:
If there is any key in either of `_chainDb` and `_stateRefDb`, wouldn't it be better to delete it?

Reply:
That way might be better, but it also might be a short moment between these things are being removed in the other thread, so I believe it's slightly better to do nothing here.

Reply:
Okay, make sense.